apps/src/netsim/NetSimPacketEditor.js

```diff
@@ -117,7 +117,7 @@ var NetSimPacketEditor = module.exports = function (initialConfig) {
    * @type {Number}
    * @private
    */
-  this.maxPacketSize_ = initialConfig.maxPacketSize || Infinity;
+  this.maxPacketSize_ = initialConfig.maxPacketSize || 8192;
```

Comment:
Obviously we don't want anything to topple over, but limiting to 128 characters when students are inventing plaintext protocols (HTML-like formatted text, for example) will be too restrictive. I'd prefer to go (safely) bigger if possible.